recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
 test:
   requires:
     - clangdev
+    - ld64
```

Comment:
Ran into [this error during testing]( https://travis-ci.org/conda-forge/openmp-feedstock/jobs/459899292#L758 ). Saw [this Clang OpenMP MacPorts issue]( https://trac.macports.org/ticket/53600 ), which notes the same issue is resolved by using `ld64` linker. So gave that a try here, but it seems it is not being used for the test. Not sure what the best way to update this test is.